debezium-core/src/main/java/io/debezium/pipeline/source/snapshot/incremental/IncrementalSnapshotSchema.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.pipeline.source.snapshot.incremental;
+
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class IncrementalSnapshotSchema {
```

Comment:
Why it is not possible to re-use standard `Table` class?

Reply:
Updated the code to use the standard Table class